attachment_preview/static/lib/ViewerJS/compatibility.js

```diff
@@ -0,0 +1,577 @@
+/* -*- Mode: Java; tab-width: 2; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
```

Comment:
This whole file loads a lot of compatibility fixes for versions of browsers and android that are not supported by recent versions of Odoo anyway, and certainly not by 14.0. So I think we should be able to get rid of this whole file.